apps/payments/next/app/[locale]/[offeringId]/[interval]/landing/route.ts

```diff
@@ -32,6 +33,8 @@ export async function GET(request: NextRequest) {
 
   let redirectUrl;
   try {
+    const metricsFlow = await getMetricsFlowAction();
```

Comment:
What happens if `getMetricsFlowAction` throws an error? Should the error be handled so that the landing page still redirects the customer or should the page crash?

Reply:
Despite its extensive use, it looks to always be an optional field. It should definitely not cause page crashes, though we should make sure we have a complete understanding of its impact as we use it further. 

An open question I have at this point is: if the flowId is created and used at one point, are there subsequent calls to FxA that will fail if the flowId is not used? OoS for this ticket I think though.

Reply:
Yup, I agree it shouldn't crash. In that case, could you wrap this call in a `try-catch`?

No impact if flowId isn't included, except for missing metrics data.

Reply:
its in a try catch along with the `await signIn` call below it, which also just logs an error when it fails (swallowing the thrown error). If you'd prefer I can bump it into its own try/catch block

Reply:
Ouff, sorry about that, I didn't see that.